Running Java benchmark MG, class D, on 1 servers (np=4), sm,self

SLURM job ID: 798165

Servers:
mpi009
mpi009
mpi009
mpi009

mpirun --mca btl_usnic_if_include 10.10.0.0/16,10.2.0.0/16 --mca btl_tcp_if_include 10.10.0.0/16,10.2.0.0/16 --mca btl sm,self --bind-to core --map-by node -np 4 /home/jsquyres/git/java-perf-test/jre1.7.0_60/bin/java -Xmx64g NPB_MPJ.MG class=D

Start: Sat Jun 21 06:49:25 PDT 2014
===============================================================================
Exception in thread "main" Exception in thread "main" Exception in thread "main" Exception in thread "main" java.lang.NegativeArraySizeException
java.lang.NegativeArraySizeException
java.lang.NegativeArraySizeException
java.lang.NegativeArraySizeException
	at NPB_MPJ.MG.runBenchmark(MG.java:198)
	at NPB_MPJ.MG.runBenchmark(MG.java:198)
	at NPB_MPJ.MG.runBenchmark(MG.java:198)
	at NPB_MPJ.MG.main(MG.java:112)	at NPB_MPJ.MG.main(MG.java:112)	at NPB_MPJ.MG.main(MG.java:112)
	at NPB_MPJ.MG.runBenchmark(MG.java:198)


	at NPB_MPJ.MG.main(MG.java:112)
-------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code.. Per user-direction, the job has been aborted.
-------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[36748,1],0]
  Exit code:    1
--------------------------------------------------------------------------
===============================================================================
End: Sat Jun 21 06:49:26 PDT 2014
